### PR TITLE
A bunch of code cleanups.

### PR DESCRIPTION
**Pull request by JakeWharton**

### CODE REVIEW

File: src/main/java/com/jakewharton/u2020/data/api/model/Repository.java [hunk 1]
```diff
@@ -48,8 +48,8 @@ private Repository(Builder builder) {
     private String name;
     private User owner;
     private String description;
-    private long stars;
-    private long forks;
+    private Long stars;
```
Comment (mattprecious) [inline src/main/java/com/jakewharton/u2020/data/api/model/Repository.java]: Why?

Comment (JakeWharton) [inline src/main/java/com/jakewharton/u2020/data/api/model/Repository.java]: Otherwise it defaults to 0 and your null check in the constructor is useless.

Comment (mattprecious) [inline src/main/java/com/jakewharton/u2020/data/api/model/Repository.java]: Hmm... can we just remove the null check? Not sure if I prefer the app crashing or defaulting to 0...

**Comment by mattprecious** [comment]
lgtm

**Pull request merged**